app/src/main/java/com/hover/stax/SplashScreenActivity.java

```diff
@@ -73,8 +128,8 @@ private void startWorkers() {
 	private void chooseNav() {
```

Comment:
Also look at Navigation Component. I've seen the app uses fragments, and this would make the navigation much smoother and easier to manage on the app

Reply:
What's happening here is deciding whether to go to fulfilRequestActivity or MainActivity. Each of those activities uses fragment components. I do think this method name needs to be updated to something better.